### ISSUE TITLE
Alignment of comment when using enumerated PERFORM

### ISSUE THREAD

**Issue by Raukze** [SAP/abap-cleaner#363]
I got a syntax error because the comment with leading `*` is not aligned at the beginning of a line.

```abap
FORM form USING  VALUE(pv_bool) TYPE abap_bool.
ENDFORM.
 
START-OF-SELECTION.
  PERFORM form USING:
* comment
       abap_true,
       abap_false.
```

```abap
FORM form USING VALUE(pv_bool) TYPE abap_bool.
ENDFORM.
 
START-OF-SELECTION.
  PERFORM form USING
       * comment
                     abap_true.
  PERFORM form USING abap_false.
```

![image001](https://github.com/user-attachments/assets/07bf217f-99a2-420b-b7b8-8ad629f30763)

Thanks in advance!

**Comment by jmgrassau** [comment]
Hi Raukze, 

please check in menu Help / About whether you are using the current release of ABAP cleaner (version 19.2) – I think this issue was already resolved. 

Kind regards,
Jörg-Michael

**Comment by jmgrassau** [comment]
P.S.: No, wait, now I can reproduce it! Okay, that's a bug, of course.

**Comment by jmgrassau** [comment]
Hi Raukze, 

this bug should be fixed with the next release. Comment lines after the colon or comma will now be moved above the unchained commands, assuming that they comment on the respective part of the chain: 

![image](https://github.com/user-attachments/assets/c5e03d2d-7beb-4e68-a3cd-12f9cec04958)

Kind regards,
Jörg-Michael

**Comment by jmgrassau** [comment]
Hi Raukze, 

thanks again for reporting this issue! I meanwhile enhanced the "stress test" (which systematically introduces comments, pragmas and colons into the ABAP cleaner test code and checks whether the cleanup rules can deal with them) and found two more cleanup rules in which this issue could have come up. All should now be fixed in version 1.19.3, which was just released!

Kind regards,
Jörg-Michael
